internal/store/group_test.go

```diff
@@ -1,22 +1,69 @@
 package store
 
 import (
+	corev1 "k8s.io/api/core/v1"
+
 	"testing"
 	"time"
 
-	"github.com/mattermost/mattermost-cloud/model"
 	"github.com/mattermost/mattermost-cloud/internal/testlib"
+	"github.com/mattermost/mattermost-cloud/model"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func TestGroups(t *testing.T) {
 	logger := testlib.MakeLogger(t)
 	sqlStore := MakeTestSQLStore(t, logger)
 
+	someBool := false
+	mattermostEnv := model.EnvVarMap{
+		"Var1": model.EnvVar{
+			Value: "Var1Value",
+			ValueFrom: &corev1.EnvVarSource{
+				FieldRef: &corev1.ObjectFieldSelector{
+					APIVersion: "1",
+					FieldPath:  "some/path/neat",
+				},
+
+				// TODO isw 2/17/2020
```

Comment:
what is `isw`?

Reply:
My initials ;)

Reply:
nice!
can you do something like
```
TODO: (ian)
```
or 
`TODO: (gigawhitlocks)`

to make explicit otherwise i will ask you again in a day, i will forgot this :joy:

Reply:
Yes absolutely! I completely understand